src/nools/task-defaults.js

```diff
@@ -0,0 +1,49 @@
+// File defining task defaults
+function taskDefaults(Utils) {
```

Comment:
Nice! Another alternative is to pass the `Utils` as parameter to `defaultResolvedIf`, then the module can return the object directly and avoid higher level variables. But then `defaultResolvedIf` would need this extra parameter every-time we call it.
Both have pros & cons and both ways work :) 
```
/// task-defaults.js
module.exports = {
{
    defaultResolvedIf: function (contact, report, event, dueDate, Utils) { 
          ...... 
    }
}

/// task-emitter.js

if (typeof taskDefinition.resolvedIf !== 'function') {
      taskDefinition.resolvedIf = function (contact, report, event, dueDate) { taskDefinition.defaultResolvedIf(contact, report, event, dueDate, Utils) }; 
     // new Javascript would be: taskDefinition.resolvedIf = (...args) => taskDefinition.defaultResolvedIf(...args, Utils)
}
```

Reply:
✨✨ best of both worlds!✨✨